samples/client/3_0_3_unit_test/python/pyproject.toml

```diff
@@ -22,12 +22,14 @@ requires-python = ">=3.8"
 dependencies = [
     "certifi >= 14.5.14",
     "immutabledict ~= 3.0.0",
+    "authlib ~= 1.2.0",
     "python-dateutil ~= 2.7.0",
     "setuptools >= 61.0",
     "types-python-dateutil",
     "types-urllib3",
     "typing_extensions ~= 4.5.0",
     "urllib3 ~= 2.0.a3",
+    "requests",
```

Comment:
Installation problems can happen when requirements are not pinned.
How about constraining this to `"requests ~= 2.31.0"`?
Note: requests depends upon urllib3

Reply:
I agree. Fixed in the new commit